docs/DeveloperGuide.md

```diff
@@ -205,7 +206,7 @@ If more than one valid prefix `-all`, `-name`, `-id`, or `stats` are present, `V
 
 Below is an example usage scenario where the `view -stats` command was entered.
 
-**Step 1:** User first calls `view -stats`. The input is passed into ``AddressBookParser`` which instantiates a ``ViewCommandParser`` instance.
+**Step 1:** User first calls `view -stats`. The input is passed into ``AddressBookParser`` which instantiates a ``1CommandParser`` instance.
```

Comment:
Hmm not sure if this is a typo but what does `1CommandParser` refer to?

Reply:
Oh my yes it's a typo xD